public/src/pages/auth/LoginNew/LoginNew.tsx

```diff
@@ -0,0 +1,91 @@
+import { AxiosRequestConfig } from 'axios';
+import React, { FC, FormEvent, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { RoutePath } from '../../../router/RoutePath';
+import { getUserData } from '../../../api/httpClient';
+import { LoginFormMode, LoginUser, UserData } from '../../../interfaces/User';
+import AuthLayout from '../AuthLayout';
+
+const defaultLoginUser: LoginUser = {
+  user: '',
+  password: '',
+  op: LoginFormMode.BASIC
+};
+
+enum RequestHeaders {
+  FORM_URLENCODED = 'application/x-www-form-urlencoded',
+  APPLICATION_JSON = 'application/json'
+}
+
+export const LoginNew: FC = () => {
+  const [form, setForm] = useState<LoginUser>(defaultLoginUser);
+  const { user } = form;
+  const [errorText, setErrorText] = useState<string | null>();
+
+  const onInput = ({ target }: { target: EventTarget | null }) => {
+    const { name, value } = target as HTMLInputElement;
+    setForm({ ...form, [name]: value });
+  };
+
+  const sendUser = (e: FormEvent) => {
+    e.preventDefault();
+    const config: Pick<AxiosRequestConfig, 'headers'> = {
+      headers: { 'content-type': RequestHeaders.FORM_URLENCODED }
+    };
+
+    getUserData(form.user, config)
+      .then((data: UserData) => {
+        return data;
+      })
```

Comment:
Remove redundant brackets
```suggestion
      .then((data: UserData) => data)
```